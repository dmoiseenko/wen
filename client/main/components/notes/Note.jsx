import React from 'react';
import PropTypes from 'prop-types';

import DeleteNoteButtonContainer from './DeleteNoteButton.container';


export default function Note({ note }) {
  return (
    <li className="block">
      <h5>{note.text} <DeleteNoteButtonContainer noteId={note.id} /></h5>
    </li>
  );
}

Note.propTypes = {
  note: PropTypes.object.isRequired
};
