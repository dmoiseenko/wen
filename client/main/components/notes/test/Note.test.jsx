import React from 'react';
import { shallow } from 'enzyme';

import Note from '../Note';


it('should render', () => {
  const props = {
    note: {
      id: 1,
      text: 'note text'
    }
  };

  const wrapper = shallow(
    <Note {...props} />
  );

  expect(wrapper.node).toMatchSnapshot();
});
