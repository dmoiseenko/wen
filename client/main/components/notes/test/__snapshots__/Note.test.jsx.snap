// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<li
  className="block"
>
  <h5>
    note text
     
    <Connect(Apollo(mapProps(DeleteNoteButton)))
      noteId={1}
    />
  </h5>
</li>
`;
